### PR TITLE
refactor: getting all headquarters using enpoint

### PR DESCRIPTION
**Pull request by jose-sal-y-rosas**
## Issue ticket number and link

https://github.com/CHUPITO-Org/FE-Chupito/issues/45

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update.

## Pull request type

Please check the type of change your PR introduces:

- [ ] :sparkles: Feature
- [ ] :bug: Bugfix
- [ ] :pencil2: Fixing typos
- [ ] :lipstick: :art: Code style update (formatting, renaming)
- [x] :recycle: Refactoring (no functional changes, no api changes)
- [ ] :zap: Performance
- [ ] :pencil: Documentation content changes
- [x] :white_check_mark: Unit testing
- [ ] :ok_hand: Updating due to code review suggest
- [ ] :rocket: Release: v
- [ ] :wrench: Add or update configuration files
- [ ] :hammer: Add or update development scripts
- [ ] :bricks: Infrastructure related changes

## What is the current module?

- Application is using firebase for getting headquarters

## What is the new behavior?

- Application is getting headquarters from API

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/pages/Events/Events.tsx [hunk 1]
```diff
@@ -26,6 +26,7 @@ export default function EventsPage(): JSX.Element {
     apiHeadquarters
       .getAll()
       .then((headquarters) => {
+        console.log('headquarters', headquarters);
```
Comment (jlora-mck) [inline src/pages/Events/Events.tsx:29]: Please remove

Comment (jose-sal-y-rosas) [reply src/pages/Events/Events.tsx]: Good catch

File: src/components/EventView/EventView.spec.tsx [hunk 2]
```diff
@@ -75,8 +75,7 @@ describe('event view component', () => {
     expect(description).toBeInTheDocument()
     expect(phoneNumber).toBeInTheDocument()
     expect(salesEventType).toBeInTheDocument()
-    expect(mockOnChange).resolves
-    expect(true).toBe(true)
+    // expect(mockOnChange).toHaveBeenCalled()
```
Comment (jlora-mck) [inline src/components/EventView/EventView.spec.tsx:78]: Also here, need to decide to include this assert or to remove it

**Review by jlora-mck — APPROVED** [review]

File: src/components/EventView/EventView.spec.tsx (hunk 2 above)
Comment (jose-sal-y-rosas) [reply src/components/EventView/EventView.spec.tsx]: done

**Review by jlora-mck — APPROVED** [review]

**Pull request merged**